src/migrations/index.js

```diff
@@ -195,6 +195,17 @@ const migrations = [
         console.log(`set sequence canned_response_id_seq to ${maxId + 1}`)
       }
     }
+  },
```

Comment:
we'll need to remove this migration with #1154 (maybe just make it a separate PR, or don't actually change old campaigns, and skip it)

Reply:
Thanks @schuyler1d, I'll remove the migration later today and push up the changes.

@ibrand I'll leave it up to you to decide whether we add the migration as a separate PR or if we just skip it. Thanks!